enhancements/declarative-index-config.md

```diff
@@ -549,21 +559,15 @@ Similarly, removing a package directory from `community-operators` and pushing t
 As a user with pull permission from the namespace an index is hosted in/as a component of OLM, I can validate the structure of the content of an index using the json representation.
  
 ```bash
-$ cd community-operators 
-$ sed -i '/defaultChannel/d' etcd/etcd.json //delete the line "defaultChannel":"singlenamespace-alpha"
+$ sed -i '/defaultChannel/d' community-operators/etcd/etcd.json //delete the line "defaultChannel":"singlenamespace-alpha"
 $ git status 
 	modified:    etcd/etcd.json
 
 $ cd ..
-$ opm index validate community-operators
-marshal error: etcd.defaultchannel: cannot convert incomplete value "string" to JSON
-$ opm index update community-operator --tag=docker.io/mynamespace/community-operators:latest
-$ opm index validate docker.io/mynamespace/community-operators:latest 
+$ opm validate community-operators
```

Comment:
I would change this `opm validate community-operators/` to make it a bit more clear that this is a directory. Just syntax sugar. Not a big deal though.